server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1091,11 +1110,13 @@ private long balanceTablets() {
     }
 
     private List<TabletMigration> checkMigrationSanity(Set<TabletServerId> current,
-        List<TabletMigration> migrations) {
+        List<TabletMigration> migrations, DataLevel level) {
       return migrations.stream().filter(m -> {
         boolean includeMigration = false;
         if (m.getTablet() == null) {
           log.error("Balancer gave back a null tablet {}", m);
+        } else if (DataLevel.of(m.getTablet().getTable()) != level) {
+          log.warn("Balancer wants to move a tablet outside of the current processing level");
```

Comment:
I think this may be too much as we actually can expect the HostRegex balancer to create migrations outside of the processing level.  Should this be debug or trace maybe?

Reply:
Updated in 967e0cd